app/assets/javascripts/index/element.js

```diff
@@ -101,4 +112,108 @@
+      action: "wbgetentities",
+      format: "json",
+      origin: "*",
+      ids: items.join("|"),
+      props: "labels|sitelinks/urls|claims|descriptions",
+      languages: langs.join("|"),
+      sitefilter: wikis.join("|")
+    }), {
+      headers: { "Api-User-Agent": "OSM-TagPreview (https://github.com/openstreetmap/openstreetmap-website)" },
+      signal: abortController?.signal
+    })
+      .then(response => response.ok ? response.json() : Promise.reject(response))
+      .then(({ entities }) => {
+        if (!entities) return Promise.reject(entities);
+        $btn
+          .closest("tr")
+          .after(
+            items
+              .filter(qid => entities[qid])
+              .map(qid => getLocalizedResponse(entities[qid]))
+              .filter(data => data.label || data.icon || data.description || data.article)
+              .map(data => renderWikidataResponse(data, $btn))
+          );
+      })
+      .catch(() => $btn.removeClass("disabled").addClass("wdt-preview"));
+  }
+
+  function getLocalizedResponse(entity) {
+    const localizedProperty = (property, langs) => langs.reduce((out, lang) => out ?? entity[property][lang], null);
+    const rank = ({ rank }) => ({ preferred: 1, normal: 0, deprecated: -1 })[rank] ?? 0;
+    const toBestClaim = (out, claim) => (rank(claim) > rank(out)) ? claim : out;
+    const data = {
+      qid: entity.id,
+      label: localizedProperty("labels", langs),
+      icon: [
+        "P8972", // small logo or icon
+        "P154", // logo image
+        "P14" // traffic sign
+      ].reduce((out, prop) => out ?? entity.claims[prop]?.reduce(toBestClaim)?.mainsnak?.datavalue?.value, null),
+      description: localizedProperty("descriptions", langs),
+      article: localizedProperty("sitelinks", wikis)
+    };
+    if (data.article) data.article.language = data.article.site.replace("wiki", "");
+    return data;
+  }
+
+  function renderWikidataResponse(data, $btn) {
+    const cell = $("<td>")
+      .attr("colspan", 2)
+      .addClass("bg-body-tertiary");
+
+    if (data.icon && OSM.WIKIMEDIA_COMMONS_URL) {
+      let src = OSM.WIKIMEDIA_COMMONS_URL + "Special:Redirect/file/" + encodeURIComponent(data.icon);
+      if (!data.icon.endsWith(".svg")) src += "?width=128";
+      $("<a>")
+        .attr("href", OSM.WIKIMEDIA_COMMONS_URL + "File:" + encodeURIComponent(data.icon))
+        .append($("<img>").attr({ src, height: "32" }))
+        .addClass("float-end mb-1 ms-2")
+        .appendTo(cell);
+    }
+    if (data.label) {
+      const label = $btn
+        .siblings(`a[href*="wikidata.org/entity/${data.qid}"]`)
+        .clone()
+        .text(data.label.value)
+        .attr("dir", "auto");
+      cell.append($("<span>").append(label));
+      if (isUnexpectedLanguage(data.label.language)) {
+        label.attr("lang", data.label.language);
+        label.after($("<sup>").text(" " + localeName.of(data.label.language)));
```

Comment:
I had to figure out what "Multiple languages" means on the third item:
<img width="318" height="260" alt="image" src="https://github.com/user-attachments/assets/6e503981-7148-40bd-8c64-e9b6d90f7185" />